packages/core/src/interfaces/charts.ts

```diff
@@ -435,6 +432,8 @@ export interface MeterChartOptions extends BaseChartOptions {
 		proportional?: {
 			total?: number
 			unit?: string
+			totalFormatter?: (total: number) => string
```

Comment:
Meter chart demos use both of these functions yet they were not established properties